tests/basic/pool_function_test.go

```diff
@@ -19,6 +19,61 @@ var (
 	storageNode   *node.Node
 	err           error
 )
+var _ = Describe("{PoolExpandMultipleTimes}", func() {
+	BeforeEach(func() {
+		contexts = scheduleApps()
+	})
+
+	JustBeforeEach(func() {
+		poolIDToResize = pickPoolToResize()
+		log.Infof("Picked pool %s to resize", poolIDToResize)
+		poolToBeResized = getStoragePool(poolIDToResize)
+	})
+
+	JustAfterEach(func() {
+		AfterEachTest(contexts)
+	})
+
+	AfterEach(func() {
+		appsValidateAndDestroy(contexts)
+		EndTorpedoTest()
+	})
+
+	It("Select a pool and expand it by 100 GiB 3 time with add-disk type. ", func() {
+		StartTorpedoTest("PoolExpandDiskAdd3Times",
+			"Validate storage pool expansion 3 times with type=add-disk", nil, 0)
+		for i := 0; i < 3; i++ {
+			poolToBeResized = getStoragePool(poolIDToResize)
```

Comment:
No, this is to get the most recent pool. 
`poolIDToResize` doesn't change but pool status does since a pool operation is triggered, therefore we need to update the objects in memory after every iteration

Reply:
got it.